fr.openfisca.org/legislation-explorer/legislation-explorer.service

```diff
@@ -2,9 +2,8 @@
 Description=OpenFisca Legislation Explorer
 
 [Service]
-Environment=NODE_ENV=production
-Environment=BASENAME=/legislation
-ExecStart=/usr/bin/node /home/openfisca/legislation-explorer/index.js
+Environment=NODE_PATH=/usr/lib/node_modules
```

Comment:
@MattiSG, any idea on how to take the `.env` conf into account on a `systemd` service?
This works, but is not fully satisfying...

Reply:
Ok, `npm start --prefix` works and is much more elegant